mitodl/hugo-course-publisher
# inacessible img

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/hugo-course-publisher/issues/132

#### What's this PR do?
Adds the inacessible gif to the project in the right place so it just works.

#### How should this be manually tested?
Launch the site, ensure that you don't see missing images where the inacessible gif should be.

#### Screenshots
![Screen Shot 2020-06-01 at 5 28 29 PM](https://user-images.githubusercontent.com/12089658/83456310-744e6600-a42d-11ea-853e-4e6a80f153a5.png)
